src/modules/app/PayloadDataType.js

```diff
@@ -6,8 +6,10 @@ import {
   EventModel,
   ExtraModel,
   LanguageModel,
-  PageModel, SprungbrettJobModel, WohnenOfferModel
+  PageModel, SprungbrettJobModel, WohnenOfferModel,
+  LocalNewsModel, TunewsModel
```

Comment:
@amusameh I think we should change to TuNewsModel to be consistent with naming on API client.